ParkingLot/ParkingLot.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="DeepEqual" Version="4.2.1" />
```

Comment:
very good, use some third-party package to solve some problem. but not check license